apps/root/src/pages/home/components/portfolio/index.tsx

```diff
@@ -170,11 +172,11 @@ const PortfolioBodyItem: ItemContent<BalanceItem, Record<string, never>> = (
       </TableCell>
       {relativeBalance !== 0 && (
         <TableCell>
-          {isLoadingPrice && !price ? (
+          {isNaN(relativeBalance) || isLoadingPrice || !price ? (
```

Comment:
I think we should catch this NaN result earlier in order to prevent it, otherwise we may show a skeleton even if there is an operation error
I'm sure this is happening because of a 0/0 operation

Reply:
agree

Reply:
The issue comes from L260 where `value.balanceUsd` is undefined, therefore it's  0 divided by `assetsTotalValue.wallet` which is also zero. Seems that this math shouldn't be done while everything is still zero. `isLoadingBalances` is false at the same time. If the new push doesn't satisfy the review, I'd just scrap it and create a linear ticket. My idea was mainly to fix visual stuff, because clearly scope will be missing a lot on these type of fixes.

Reply:
@FiboApe @th0rOdinson  wdyt?

Reply:
Left one comment related 😄